app/models/patient.rb

```diff
@@ -28,10 +28,6 @@ class Patient < ApplicationRecord
     in: VALID_PATIENT_ENUMS[:monitoring_plan],
     message: "is not an acceptable value, acceptable values are: '#{VALID_PATIENT_ENUMS[:monitoring_plan].reject(&:blank?).join("', '")}'"
   }
-  validates :exposure_risk_assessment, inclusion: {
-    in: VALID_PATIENT_ENUMS[:exposure_risk_assessment],
-    message: "is not an acceptable value, acceptable values are: '#{VALID_PATIENT_ENUMS[:exposure_risk_assessment].reject(&:blank?).join("', '")}'"
-  }
```

Comment:
@ashankland ?

Reply:
I'm guessing it might of been removed as a result of trying to deal with a merge conflict or something, will add this back for now unless @ashankland mentions a reason

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/689/commits/a64587cc4e781563136ee93f313d4d74d696d7fb#diff-0b38c12b7f12dd9f316a576d5b5dc6c12f0b8019720598550b239fca8828d292R57

Reply:
addressed here: https://github.com/SaraAlert/SaraAlert/pull/689/commits/a64587cc4e781563136ee93f313d4d74d696d7fb?file-filters%5B%5D=.rb#diff-94dcc014f62f0d9b6084ad98f82f74cd40c86b029ef1691914bdbc960c2f42dcR32